app/models/edition/topical_events.rb

```diff
@@ -22,13 +22,10 @@ def can_be_associated_with_topical_events?
   end
 
   def search_index
-    # DID YOU MEAN: Policy Area?
     # "Policy area" is the newer name for "topic"
     # (https://www.gov.uk/government/topics)
-    # "Topic" is the newer name for "specialist sector"
-    # (https://www.gov.uk/topic)
-    # You can help improve this code by renaming all usages of this field to use
-    # the new terminology.
-    super.merge("topics" => topical_events.map(&:slug)) {|k, ov, nv| ov + nv}
```

Comment:
Presumably "old value" and "new value". This form of `#merge` passes both through to the block and then does the merge. I don't find it readable either so I've just rewritten it using the block-less version of merge.